### PR TITLE
Version 3.2.0

### PR DESCRIPTION
**Pull request by Breaka84**
3.2.0 (2021-04-13)
-------------------
* [MOD] add functionality to log cleansed values into separate struct column (column_to_log_cleansed_values)
* [MOD] add ignore_ambiguous_columns to Mapper
* [MOD] log spooq version when importing
* [REM] Drop separate spark package (bin-folder) as pip package can now handle all tests as well
* [ADD] Github action to test on label (test-it) or merge into master

### CODE REVIEW

**Comment by Breaka84** [comment]
I pushed the flattener stuff to a future version to get the cleansing logging in the code

File: src/spooq2/transformer/base_cleaner.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+from builtins import object
+import logging
```
Comment (rt-phb) [inline src/spooq2/transformer/base_cleaner.py:2]: you can remove those two imports, imports already happen in the super class

File: src/spooq2/transformer/base_cleaner.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+from pyspark.sql import functions as F
+
+from .transformer import Transformer
+
+
+class BaseCleaner(Transformer):
+
```
Comment (rt-phb) [inline src/spooq2/transformer/base_cleaner.py:9]: maybe add one line of documentation

**Comment by rt-phb** [comment]
rest looks good! everything is working. did also some manual testing

**Review by rt-phb — APPROVED** [review]

**Pull request merged**